src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -116,7 +116,8 @@
 <kd-help-section>
   <div class="md-block">
     <md-checkbox ng-model="ctrl.isExternal" class="md-primary"
-                 ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }">
+                 ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }"
+                 ng-disabled="!ctrl.isExposeEnabled()">
```

Comment:
Ok, I just noticed. The model property is called isExternal, so I think the validation check should be called isExternalEnabled